msteams-platform/bots/how-to/authentication/bot-sso-code.md

```diff
@@ -475,9 +475,9 @@ Use the following code snippet to handle the access token in case the app user l
 
 ## Code sample
 
-| **Sample name** | **Description** | **C#** | **JavaScript** | **Node.js** |
-| --- | --- | --- | --- | --- |
-| Bot framework SDK | This sample code demonstrates how to get started with authentication in a bot for Microsoft Teams. | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/bot-conversation-sso-quickstart/csharp_dotnetcore/BotConversationSsoQuickstart) | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/bot-conversation-sso-quickstart/js)  | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/bot-conversation-sso-quickstart/js) |
+| **Sample name** | **Description** | **C#** | **Node.js** |
+| --- | --- | --- | --- | 
+| Bot framework SDK | This sample code demonstrates how to get started with authentication in a bot for Microsoft Teams. | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/bot-conversation-sso-quickstart/csharp_dotnetcore/BotConversationSsoQuickstart) | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/bot-conversation-sso-quickstart/js)  |
```

Comment:
Start with a standard word for all the samples like: This sample shows the

Reply:
Changed